test/unit_test/runtime/enzimology_test.cpp

```diff
@@ -0,0 +1,99 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2023 Meskauskas Audrius
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in all
+// copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+// SOFTWARE.
+
+#include "almost_equals.h"
+#include <catch2/catch_all.hpp>
+#include <mp-units/systems/enzimology/enzimology.h>
+#include <mp-units/systems/isq/space_and_time.h>
+#include <mp-units/systems/si/unit_symbols.h>
+#include <mp-units/systems/si/units.h>
+#include <limits>
+
+using namespace mp_units;
+using namespace mp_units::si::unit_symbols;
+using namespace mp_units::si;
+
+// Check agreements between international unit (iu), unit and katal
+static_assert(1. * enzimology::unit == 1. * enzimology::iu);
+static_assert(1. * enzimology::unit < 16.6667 * enzimology::nanokatal);
+static_assert(1. * enzimology::unit > 16.6666 * enzimology::nanokatal);
+
+TEST_CASE("Enzimology calcuclations", "[enzimology]")
+{
+  // In the test tube with 2 ml of test solution, 10 micro-molles of the substrate have been processed in 5 minutes.
+  SECTION("calculations")
+  {
+    const auto ml = milli<litre>;
+    const auto mM = milli<mole>;
+    const auto mkM = micro<mole>;
```

Comment:
Header `<mp-units/systems/si/unit_symbols.h>`
already has `mmol` and `umol`,
which are correctly named.

Reply:
Corrected, thanks. All tests still pass.

Reply:
That's good!
I'm not too experienced with V2,
so I wasn't sure about anything.

Reply:
We learn by doing... 😉